.github/workflows/submit.yml

```diff
@@ -13,8 +13,6 @@ jobs:
       collection: '2025'
       kind: original
       path: papers/*
-      label: publish
-      publish: true
```

Comment:
I removed `publish` from here - If present, it will publish a new version to the site without any additional review (as opposed to creating a submission which you then have to publish separately).

If you do want this automatic publishing on `main`, feel free to add this back.